catmap/scalers/generalized_linear_scaler.py

```diff
@@ -431,7 +431,7 @@ def parse_constraints(self,constraint_dict):
             for ads in self.parameter_names:
                 if ads not in constraint_dict:
                     constr = self.default_constraints
-                elif not hasattr(constraint_dict[ads], '__iter__'):
```

Comment:
This also needs to be done in lines [49](https://github.com/mhoffman/catmap/blob/bcb884136dcde4d3c64020410d3b407a9864e10c/catmap/scalers/generalized_linear_scaler.py#L49), [87](https://github.com/mhoffman/catmap/blob/bcb884136dcde4d3c64020410d3b407a9864e10c/catmap/scalers/generalized_linear_scaler.py#L87), [176](https://github.com/mhoffman/catmap/blob/bcb884136dcde4d3c64020410d3b407a9864e10c/catmap/scalers/generalized_linear_scaler.py#L176) and [282](https://github.com/mhoffman/catmap/blob/bcb884136dcde4d3c64020410d3b407a9864e10c/catmap/scalers/generalized_linear_scaler.py#L282)